### PR TITLE
Enikshay: Added new fields to sputum conversion report

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by czue** [comment]
tests are failing
```
======================================================================
FAIL: test_extra_pulmonary (custom.enikshay.ucr.tests.test_episode_datasource.TestEpisodeDatasource)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 222, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 195, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 210, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 191, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/custom/enikshay/ucr/tests/test_episode_datasource.py", line 189, in test_extra_pulmonary
    self.assertEqual(row.new_smear_positive_extra_pulmonary_TB_male, 1)
AssertionError: 0 != 1
```

**Review by czue — APPROVED** [review]
not reviewing closely but looks good on :white_check_mark:

**Comment by kkrampa** [comment]
@czue

File: custom/enikshay/ucr/data_sources/test.json [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+                        }
+                    },
+                    "sort_expression": {
+                        "type": "property_name",
+                        "property_name": "opened_on"
+                    }
+                },
+                "aggregation_fn": "last_item"
+            },
+            "complete_treatment_form": {
```
Comment (czue) [inline custom/enikshay/ucr/data_sources/test.json]: :dizzy_face:

**Review by czue — APPROVED** [review]

**Comment by czue** [comment]
thanks @kkrampa !

**Pull request merged**